lib/Pipfile

```diff
@@ -68,3 +69,6 @@ numpy = "*"
 futures = "*"
 pandas = "<0.25.0"
 numpy = "<1.17.0"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Why do we need this? 

I think this will make all `*` versions possibly be pre-release. Sounds like something we should avoid, if at all possible